src/ansys/fluent/core/session_solver.py

```diff
@@ -116,36 +116,54 @@ def _build_from_fluent_connection(self, fluent_connection):
         else:
             self.reduction = reduction_old
         self._settings_api_root = None
+        self.fields.solution_variable_data = self._solution_variable_data()
 
     def build_from_fluent_connection(self, fluent_connection):
         """Build a solver session object from fluent_connection object."""
         super(Solver, self).build_from_fluent_connection(fluent_connection)
         self._build_from_fluent_connection(fluent_connection)
 
-    @property
-    def solution_variable_data(self) -> SolutionVariableData:
+    def _solution_variable_data(self) -> SolutionVariableData:
         """Return the SolutionVariableData handle."""
         return service_creator("svar_data").create(
-            self._solution_variable_service, self.solution_variable_info
+            self._solution_variable_service, self.fields.solution_variable_info
         )
 
     @property
     def svar_data(self):
         """Return the SolutionVariableData handle."""
         warnings.warn(
-            "svar_data is deprecated, use solution_variable_data instead",
+            "svar_data is deprecated, use fields.solution_variable_data instead",
```

Comment:
We could presumably remove the warnings at the `svar_*` level now. We will get them automatically next level down.

Reply:
@seanpearsonuk, I will remove the warning from solution_variable_* for the reason mentioned.